common/hashicorp-vault/values.yaml

```diff
@@ -12,18 +12,7 @@ vault:
     extraEnvironmentVars:
       VAULT_CACERT: /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
       VAULT_ADDR: https://vault.vault.svc.cluster.local:8200
-    standalone:
```

Comment:
I think we still want the standalone config key? (It's still referenced in the overrides below). We can still use `{{ .vault.server.standalone }}` when overridden appropriately. It seems like the main differences is `disable_mlock` which we might be able to figure something out something clever to do with.

Reply:
Reverted as advised